From d484b576e960a216ec189b9ebe1967ee21a340f8 Mon Sep 17 00:00:00 2001
From: Colin Watson <cjwatson@ubuntu.com>
Date: Sun, 9 Feb 2014 12:58:24 -0600
Subject: Fix test-suite failure caused by formfeed differences in texinfo 5.

The version of makeinfo in texinfo 5.2 (at least) emits
slightly-differently-formatted output that breaks
ice-9/documentation.scm, and hence the test suite.  Later versions of
Guile implement their own texinfo parsing and thus no longer rely on
makeinfo for this, but this workaround is enough to fix the test suite
for now.

[rlb@defaultvalue.org: adjust commit message.]

Closes: #711029
Bug-Debian: http://bugs.debian.org/711029
---
 libguile/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libguile/Makefile.am b/libguile/Makefile.am
index 1299cf2..e2e8025 100644
--- a/libguile/Makefile.am
+++ b/libguile/Makefile.am
@@ -304,7 +304,7 @@ if HAVE_MAKEINFO
 
 guile-procedures.txt: guile-procedures.texi
 	rm -f $@
-	makeinfo --force -o $@ guile-procedures.texi || test -f $@
+	makeinfo --force --no-headers guile-procedures.texi | sed 's/  *//g' > $@
 
 else
 
